shared/utils/parseDocumentSlug.ts

```diff
@@ -13,11 +13,11 @@ export default function parseDocumentSlug(url: string) {
     try {
       parsed = new URL(url).pathname;
     } catch (err) {
-      return;
+      return "";
```

Comment:
Returning `""` instead of `undefined` because it helps in dealing with nasty type errors. I don't think `undefined` here is particularly used for any specific purpose and as far as _falsy_ tests are concerned, `""` equally works.

Reply:
I think it's useful in that it stops you in making a useless request to `document.users` with an empty id. Instead we can guard against that and make no request at all of this returns `undefined`?

Reply:
Better :+1: